File: app/src/main/java/com/my/televip/virtuals/ActionBar/ActionBarMenuItem.java
package com.my.televip.virtuals.ActionBar;

import com.my.televip.obfuscate.ArgsResolver;
import com.my.televip.obfuscate.Obfuscate;

import de.robv.android.xposed.XposedHelpers;

public class ActionBarMenuItem {

    Object actionBarMenuItem;

    public ActionBarMenuItem(Object obj){
        actionBarMenuItem = obj;
    }

    public void lazilyAddSubItem(int id, int resId, String name){
        XposedHelpers.callMethod(actionBarMenuItem, Obfuscate.getMethodName("ActionBarMenuItem", "lazilyAddSubItem"), id, resId, name);
    }
    public void addSubItem(int id, int resId, String name){
        XposedHelpers.callMethod(actionBarMenuItem, Obfuscate.getMethodName("ActionBarMenuItem", "addSubItem"), id, resId, name);
    }

    public Object getActionBarMenuItem(){
        return  actionBarMenuItem;
    }
}

File: app/src/main/java/com/my/televip/virtuals/ActionBar/ActionBarMenuItem.kt
package com.my.televip.virtuals.ActionBar

import com.my.televip.obfuscate.Obfuscate
import de.robv.android.xposed.XposedHelpers

class ActionBarMenuItem(@JvmField val actionBarMenuItem: Any?) {

    fun lazilyAddSubItem(id: Int, resId: Int, name: String?) {
        val target = actionBarMenuItem ?: return
        XposedHelpers.callMethod(target, Obfuscate.getMethodName("ActionBarMenuItem", "lazilyAddSubItem"), id, resId, name)
    }

    fun addSubItem(id: Int, resId: Int, name: String?) {
        val target = actionBarMenuItem ?: return
        XposedHelpers.callMethod(target, Obfuscate.getMethodName("ActionBarMenuItem", "addSubItem"), id, resId, name)
    }
}
